src/main/java/org/wildfly/security/password/impl/OneTimePasswordAlgorithmParametersSpiImpl.java

```diff
@@ -54,7 +54,7 @@ protected void engineEncode(final ASN1Encoder encoder, final OneTimePasswordAlgo
     protected OneTimePasswordAlgorithmSpec engineDecode(final ASN1Decoder decoder) {
         decoder.startSequence();
         final String algorithm = decoder.decodeOctetStringAsString();
-        final byte[] seed = decoder.decodeOctetString();
+        final String seed = decoder.decodeOctetStringAsString();
```

Comment:
Actually, after taking a closer look at this, since the seed must consist of strictly alphanumeric characters from the ISO-646 Invariant Code Set, I think it makes more sense to use encodeIA5String and decodeIA5String instead of encodeOctetString and decodeOctetStringAsString when DER encoding and decoding the seed in OneTimePasswordAlgorithmParametersSpiImpl and KeyStoreCredentialStore.